app/routes.py

```diff
@@ -1 +1,244 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, jsonify, request, abort, make_response
+from app import db
+from app.models.task import Task
+from datetime import datetime
+from app.models.goal import Goal
+import os
+import requests
+
+tasks_bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+
+def get_one_model_or_error(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except ValueError:
+        response_body = "Invalid data"
+        abort(make_response(jsonify({"details": response_body}), 400))
+
+    selected_obj = cls.query.get(model_id)
+
+    if selected_obj is None:
+        response_body = "Not found"
+        abort(make_response(jsonify({"message": response_body}), 404))
+
+    return selected_obj
+
+@tasks_bp.route("", methods=["POST"])
+def add_task():
+
+    request_body = request.get_json()
+
+    if "title" not in request_body or \
+        "description" not in request_body:
+        return jsonify({"details": "Invalid data"}), 400
+
+    new_task = Task.from_dict(request_body)
+
+    db.session.add(new_task)
+    db.session.commit()
+
+    response = Task.to_dict(new_task)
+
+    return jsonify({"task": response}), 201 
+
+
+@tasks_bp.route("", methods=["GET"])
+def get_all_tasks():
+
+    sort_params = request.args.get("sort")
+
+    tasks = Task.query.all()
+
+    response = []
+
+    for task in tasks:
+        task_dict = {
+            "id": task.task_id,
+            "title": task.title,
+            "description": task.description,
+            "is_complete": False
+        }
+        response.append(task_dict)
+
+    if sort_params == "asc":
+        response = sorted(response, key=lambda task: task['title'])
+    elif sort_params == "desc":
+        response = sorted(response, key=lambda task: task['title'], reverse=True)
```

Comment:
This works, but it's more efficient to use Postgres to do the sorting for you, SQL Alchemy has some methods to do this.  

https://docs.sqlalchemy.org/en/14/orm/query.html#sqlalchemy.orm.Query.order_by